content/derisking-government-tech/about.md

```diff
@@ -3,7 +3,33 @@ title: About
 permalink: /derisking-government-tech/about/
 layout: layouts/page
 tags: derisking-government-tech
+description: History and authors of the 18F De-risking Government Technology Guide 2.0.
 sidenav: false
 ---
 
-TK
+{% include 'derisking-government-tech/chapter_image.html' img_path: "assets/derisking-government-tech/img/icon.svg" %}
+
+The De-risking Government Technology Guide 2.0 consolidates content from the two parts of the original De-risking Guide: the [State Software Budgeting Handbook]({{ "https://guides.18f.gov/derisking/state-field-guide/" | url }}), released in August 2019, and the [Federal Agency Field Guide]({{ "https://guides.18f.gov/derisking/federal-field-guide/" | url }}), released in September 2020. In addition, the new section “[Working with a vendor development team]{{ "/derisking-government-tech/vendor-management/" | url }}” includes detailed guidance about vendor management. 
```

Comment:
change: Internal links should only link to paths, `{{ "/derisking/state-field-guide/" | url }}`